src/neynar-api/v2/openapi-farcaster/apis/block-api.ts

```diff
@@ -24,13 +24,56 @@ import { BASE_PATH, COLLECTION_FORMATS, type RequestArgs, BaseAPI, RequiredError
 // @ts-ignore
 import type { BlockListResponse } from '../models';
 // @ts-ignore
+import type { BlockReqBody } from '../models';
+// @ts-ignore
 import type { ErrorRes } from '../models';
+// @ts-ignore
+import type { OperationResponse } from '../models';
 /**
  * BlockApi - axios parameter creator
  * @export
  */
 export const BlockApiAxiosParamCreator = function (configuration?: Configuration) {
     return {
+        /**
+         * Adds a block for a given FID.
+         * @summary Block FID
+         * @param {BlockReqBody} blockReqBody 
+         * @param {*} [options] Override http request option.
+         * @throws {RequiredError}
+         */
+        addBlock: async (blockReqBody: BlockReqBody, options: RawAxiosRequestConfig = {}): Promise<RequestArgs> => {
```

Comment:
Need OAS change here
`operationId` should be kebak-case and match the name of the method.
`publish-block`